### PR TITLE
Add cos config in aws sdk conf

### PR DESCRIPTION
**Pull request by stephen-shelby**

### CODE REVIEW

**Review by DorianZheng — APPROVED** [review]
LGTM

**Comment by miomiocat** [comment]
lgtm

File: using_starrocks/External_table.md [hunk 1]
```diff
@@ -299,9 +299,11 @@ select count(*) from profile_wos_p7;
   3. 在$FE_HOME/conf/fe.conf 文件中的 JAVA_OPTS/JAVA_OPTS_FOR_JDK_9 选项加上 -Djava.security.krb5.conf:/etc/krb5.conf，/etc/krb5.conf 是 krb5.conf 文件的路径，可以根据自己的系统调整。
   4. resource 中的 uri 地址一定要使用域名，并且相应的 hive 和 hdfs 的域名与 ip 的映射都需要配置到/etc/hosts 中。
 
-#### S3 支持
+#### AWS S3/Tencent Cloud COS支持
 
-一. 在 $FE_HOME/conf/core-site.xml 中加入如下配置。
+一. 下载[依赖库](https://cdn-thirdparty.starrocks.com/hive_s3_jar.tar.gz)并添加到fe/lib/和be/lib/hadoop/hdfs/路径下。
```
Comment (DorianZheng) [inline using_starrocks/External_table.md:304]: be不需要了吧

Comment (stephen-shelby) [reply using_starrocks/External_table.md]: done

**Review by Dshadowzh — APPROVED** [review]
LGTM

**Comment by stephen-shelby** [comment]
@Mergifyio backport 2.2

**Pull request merged**